fuse_models/src/acceleration_2d.cpp

```diff
@@ -71,6 +70,10 @@ void Acceleration2D::onInit()
     ROS_WARN_STREAM("No dimensions were specified. Data from topic " << ros::names::resolve(params_.topic) <<
                     " will be ignored.");
   }
+
+  if (!params_.target_frame.empty()) {
```

Comment:
I think the coding style should be like this
```suggestion
  if (!params_.target_frame.empty())
  {
```